ethereum/circuits/lib/Nargo.toml

```diff
@@ -1,7 +1,6 @@
 [package]
 name = "ethereum"
 type = "lib"
-authors = ["Arkadiusz Konior, Marek Kirejczyk"]
```

Comment:
Or should we mention vlayer team or just name all commiters?

Reply:
Naming all commiters is is a PITA as people join. And I don't see a point. We can write "vlayer" there, but it's already in the vlayer repo, it will be published by our account and in our namespace. Don't see a point of this field except for when the library is authored by one or two authors. I think it makes sense when you list e-mails of the authors so that you can e-mail them about issues, but it's some 1992 thinking